src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/training
+spring.datasource.username= ana_java
```

Comment:
This is just a comment for the future, be careful not to push credentials or sensitive information to the repository (in general they are set up as secrets in the deployment environment, and as environment variables in the IDE). For the purposes of the training this is fine, you don't need to change it, but it's just a tip for future projects  😄

Reply:
Yes, I know :D 
I didn't know how it was handled in Java, so I pushed it precisely to ask about it since it was only mocked data.

Reply:
Awesome! 😄